recipes/label_parsing.ipynb

```diff
@@ -0,0 +1,3063 @@
+{
```

Comment:
<p>Will you be adding this tutorial to the TOC listing tutorials in SDK docs? If so, maybe we should also add something to the intro topic that I sent you today?</p><p>Also: how about adding a summary list of items covered in this NB? Other NBs see to have one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>